core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -22,14 +22,14 @@
 import org.apache.lucene.search.spell.StringDistance;
 import org.apache.lucene.search.spell.SuggestMode;
 import org.apache.lucene.util.automaton.LevenshteinAutomata;
+import org.elasticsearch.search.suggest.term.TermSuggestionBuilder;
 
 public class DirectSpellcheckerSettings  {
 
     // NB: If this changes, make sure to change the default in TermBuilderSuggester
     public static SuggestMode DEFAULT_SUGGEST_MODE = SuggestMode.SUGGEST_WHEN_NOT_IN_INDEX;
     public static float DEFAULT_ACCURACY = 0.5f;
-    // NB: If this changes, make sure to change the default in TermBuilderSuggester
-    public static Suggest.Suggestion.Sort DEFAULT_SORT = Suggest.Suggestion.Sort.SCORE;
+    public static TermSuggestionBuilder.SortBy DEFAULT_SORT = TermSuggestionBuilder.SortBy.SCORE;
```

Comment:
I just checked and to me it looks safe to replace `Suggest.Suggestion.Sort` since it used only for a response object but would like @areek to take a second look. Also wondering if putting the `SortBy` enum in its own class (maybe `SuggestionSort` would make the intend a bit cleaner. I somehow don't like this enum beeing inner class of TermSuggestionBuilder, because it seems a general sort for suggestions. Maybe anyone else has opinions?